utils/examples/generate-examples.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import os
+import sys
+import pprint
+import logging
+
+import verovio
+
+IMAGES_PATH = "../../build/assets/images/GeneratedImages"
+
+VRV_OPTIONS: dict = {
+   'scale': 10,
+   'pageWidth': 1500,
+   'adjustPageHeight': True,
+   'adjustPageWidth': True,
+   'footer': 'none',
+   'header':'none',
+   'openControlEvents': True,
+   'outputFormatRaw': True,
+   'removeIds': True,
+   'svgFormatRaw': True,
+   'svgHtml5': True,
+   'svgRemoveXlink': True,
+   'svgViewBox': True,
+    'xmlIdSeed': 1
+}
+
+success: bool = True
+
+if __name__ == "__main__":
+    description = """
+        Renders SVG for the MEI examples. Will generate any new examples found; to
+        re-generate all examples use the '--clean' option.
+    """
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument("--clean", action="store_true", help="Re-generate all examples.")
+    verbose_group = parser.add_mutually_exclusive_group()
+    verbose_group.add_argument("--verbose", "-v", action="store_true")
+    verbose_group.add_argument("--debug", "-d", action="store_true")
+
+    args = parser.parse_args()
+    if args.debug:
+        level_msg: str = "Debug"
+        level = logging.DEBUG
+    elif args.verbose:
+        level_msg: str = "Info"
+        level = logging.INFO
+    else:
+        level_msg: str = "Error"
+        level = logging.ERROR
+
+    logging.basicConfig(format="[%(asctime)s] [%(levelname)8s] %(message)s (%(filename)s:%(lineno)s)",
+                        level=level)
+    log = logging.getLogger(__name__)
+    log.info("Running at logging level %s", level_msg)
+
+    verovio.enableLog(verovio.LOG_DEBUG)
+    tk = verovio.toolkit()
+    # version of the toolkit
+    log.info("Using Verovio %s", tk.getVersion())
+
+    log.debug("Running Verovio with the following options: %s", pprint.pformat(VRV_OPTIONS))
+    tk.setOptions(VRV_OPTIONS)
+
+    for file in os.listdir(IMAGES_PATH):
+        # Skip everything that is not an .mei or .xml file
+        if not file.endswith(".mei") and not file.endswith(".xml"): 
```

Comment:
I would prefer to clean up the files and having a unified extension.

Reply:
Sounds good. They are all `.mei`, I think.

Reply:
Yes, they are all `.mei`. Checking for only that extension should be fine.